spec/integration/admin/activity_insight_oa_workflow/wrong_file_version_review_spec.rb

```diff
@@ -69,7 +69,9 @@
 
   describe 'clicking the button to send a single email notification' do
     it 'sends an email and displays a confirmation message' do
+      sleep(2)
```

Comment:
If the sleep here did fix the test, then I think `sleep(0.5)` will probably be sufficient.  Also, I think the second sleep below can be removed.  At that point capybara is just using whatever is stored in `inputs` to determine what to click, so there shouldn't be anymore delays causing trouble.

Reply:
I don't think the sleep is going to be reliable.  I managed to wrack my brain enough to put together an algorithm that should get it right every time:

`find_all("input[name='publications']", visible: false).collect{|i| i if i.value == pub1.id.to_s}.compact.first.sibling("input").click`

Reply:
Or as rubocop prefers:

`find_all("input[name='publications']", visible: false).select { |i| i.value == pub1.id.to_s }.first.sibling('input').click`